### DISCUSSION TITLE
knobs area

### DISCUSSION THREAD

**Discussion by AndersonOli** [ookami-kb/storybook_flutter#35] [Ideas]
Hi folks!

It's possible provide a child to knob_panel instead of the standard injection?

Imagina that I want a custom state to my widget, I'll not have this with knob builder text or boolen methods;

if I did a pull request for this, would this idea be acceptable?

**Comment by ookami-kb** [comment]
Hi,

We don't have some use cases for complex knobs - for us it's mainly about enabling/disabling some features, so boolean is the most used one.

Do you have some specific use case in mind as an example?

In general, I think this feature can definitely be useful, so your contribution is welcome.
